simulation/adx_gyro_accelerometer/src/main/native/cpp/ADXRS450_SpiGyroWrapperData.cpp

```diff
@@ -85,11 +89,10 @@ void ADXRS450_SpiGyroWrapper::HandleAutoReceiveData(uint8_t* buffer,
 
   while (msgCtr < valuesToRead) {
     double cappedDiff = diff;
-    if (cappedDiff > kMaxAngleDeltaPerMessage) {
-      cappedDiff = kMaxAngleDeltaPerMessage;
-    } else if (cappedDiff < -kMaxAngleDeltaPerMessage) {
-      cappedDiff = -kMaxAngleDeltaPerMessage;
-    }
+
+    // Clamp the value to the max you can send in one packet
+    cappedDiff = std::max(cappedDiff, -kMaxAngleDeltaPerMessage);
```

Comment:
Instead of having a comment saying the value is clamped, why not just write the constexpr template function called clamp()? It expresses the intent much more clearly. I guess we could put the definition in this file for now the same way we do for PIDController.cpp.